EIPS/eip-2681.md

```diff
@@ -1,6 +1,7 @@
 ---
 eip: 2681
 title: Limit account nonce to 2^64-1
+description: Limit account nonce to be between `0` and `2^64-1`(inclusive).
```

Comment:
The `inclusive` wording is materially incorrect.

The specification states:
> 1. Consider any transaction invalid, where the nonce exceeds or equals to `2^64-1`.

Reply:
I am such an idiot....